GimiFeedback/GimiFeedback/Presentation/FeedbackList/FeedbackListView.swift

```diff
@@ -1,10 +1,3 @@
-//
-//  FeedbackListView.swift
-//  GimiFeedback
-//
-//  Created by 김민석 on 7/8/25.
-//
-
 import SwiftUI
 
 struct FeedbackListView: View {
```

Comment:
이부분 Name 혹시 
```suggestion

struct FeedbackChannelListView: View {
```

은 어떠신가요?

Reply:
해당view 관련해서는 이미 FeedbackListView가 만들어져 있어서 그것을 기준으로 작업했습니다.
파일 이름(view) 같은거는 저희가 이미 피쳐들이 거의다 나와서 다 같이 정해서 진행하면 훨씬 편할 것 같습니다.
어떻게 보면 해당 페이지가 앱 실행후 (로그인 했다고 가정) 첫 페이지 이므로 Home이나 Dashboard로 표현이 될 것도 같습니다.